google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.getImageURI());
```

Comment:
Agreed. I don't like this approach. But I don't understand your comment about making a getter, either. The getter would still need to be accessed and cannot be "notified", right? The whole point of this CL was to address upward data binding. Is that possible with a getter?
However, as an aside, it would be better if `getImageURI` was a getter instead of a method.